src/app-layout/visual-refresh-toolbar/exports.tsx

```diff
@@ -1,7 +1,10 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-export { AppLayoutDrawerImplementation as AppLayoutDrawer } from './drawer';
+export {
+  AppLayoutDrawerImplementation as AppLayoutDrawer,
+  AppLayoutGlobalDrawersImplementation as AppLayoutGlobalDrawers,
```

Comment:
@georgylobko note, I already used the plural name `GlobalDrawers` according to my comment on the main PR